app/src/core/components/ColorModeToggle.tsx

```diff
@@ -2,13 +2,10 @@ import Brightness4Icon from '@mui/icons-material/Brightness4';
 import Brightness7Icon from '@mui/icons-material/Brightness7';
 import IconButton from '@mui/material/IconButton';
 import { SvgIconPropsColorOverrides } from '@mui/material/SvgIcon/SvgIcon';
-import { useTheme } from '@mui/material/styles';
+import { useColorScheme } from '@mui/material/styles';
 import { OverridableStringUnion } from '@mui/types';
-import { useContext } from 'react';
 import styled from 'styled-components';
 
-import { ColorModeContext } from '~/core/stores/themeMode';
```

Comment:
For the template do we want to ditch the light/dark mode items we had? They were pulled in PLUT only because it has no dark mode and I don't see a realistic plan for it to have one anytime soon

Reply:
Good call - I guess we should probably keep that